rllib/core/models/catalog.py

```diff
@@ -224,3 +240,80 @@ def get_tokenizer_config(
                 **{"use_lstm": False, "use_attention": False},
             },
         )
+
+    @classmethod
+    def get_action_dist_cls_dict(
+        cls,
+        action_space: gym.Space,
+        model_config_dict: ModelConfigDict,
+        deterministic: Optional[bool] = False,
+    ) -> Mapping[str, Any]:
+        """Returns a mapping from framework to distribution class.
+
+        You can get the required input dimension for the distribution by calling
+        `action_dict_cls.required_model_output_shape(action_space, model_config_dict)`
+        on the retrieved class. This is useful, because the Catalog needs to find out
+        about the required input dimension for the distribution before the model that
+        outputs these inputs is configured.
+
+        Args:
+            action_space: Action space of the target gym env.
+            model_config_dict: Optional model config.
+            deterministic: Whether to return a Deterministic distribution instead of
+                a diagonal Gaussian one for Box spaces.
+
+        Returns:
+                Mapping from framework to distribution class.
+        """
+        assert not model_config_dict.get("custom_action_dist"), (
+            "Specify a custom action "
+            "distribution by overwriting "
+            "Catalog.get_action_distribution."
+        )
+
+        # Box space -> DiagGaussian OR Deterministic.
+        if isinstance(action_space, Box):
+            if action_space.dtype.name.startswith("int"):
```

Comment:
why can't we just check `isinstance(action_space, MultiDiscrete)`?

Reply:
Sorry, I understood the condition now. Isn't there any better api to check the dtype? I particularly don't feel safe about this `.name.startswith("int"):`

Reply:
Yeah it's wonky, you are right. We had the same logic in the old catalog. I guess we should go with the "let's not try to be clever" logic here and simply raise an error that tells folks to use a fitting distribution.

Reply:
I don't think there is a standard API, but `action_space.dtype.char in np.typecodes["AllInteger"]` seems like a stable and precise solution.